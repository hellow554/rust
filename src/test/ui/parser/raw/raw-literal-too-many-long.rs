fn main() {
    let a = r##"This //~ ERROR too many `#` when terminating raw string
    is
    a
    very
    long
    string
    which
    goes
    over
    a
    b
    c
    d
    e
    f
    g
    h
    lines
    "###;
}
