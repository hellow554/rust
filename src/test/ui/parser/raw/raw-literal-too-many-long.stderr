error: too many `#` when terminating raw string
  --> $DIR/raw-literal-too-many-long.rs:2:13
   |
LL |       let a = r##"This
   |               ^-- this raw string has 2 leading `#`...
   |  _____________|
   | |
LL | |     is
LL | |     a
LL | |     very
...  |
LL | |     lines
LL | |     "###;
   | |______--^
   |        |
   |        ...but is closed with 3.
   = help: remove the unneeded `#`

error: aborting due to previous error

